app/src/main/java/org/pesmypetcare/mypetcare/controllers/ControllersFactory.java

```diff
@@ -12,4 +12,25 @@ public static TrRegisterNewPet createTrRegisterNewPet() {
     public static TrUpdatePetImage createTrUpdatePetImage() {
```

Comment:
This class will be commented whenever all controllers of the iteration are included